process/tac-member-R&Rs.md

```diff
@@ -0,0 +1,13 @@
+# TAC Member Roles & Responsibilities
+
+- Commit to abiding by the OpenSSF Code of Conduct
```

Comment:
While I certainly agree with the statement it applies to everybody engaging in OpenSSF, not just the TAC. Maybe it's better to leave it off and keep this list to what is specific to TAC members?

Reply:
@presidentoor do you want to weigh in here?

Reply:
I think the language should read: "Abide by the OpenSSF Code of Conduct"

Reply:
Ok but the question is whether we should have any such language here at all. I think not unless we go and add it everywhere else.

Reply:
Ok but the question is whether we should have any such language here at all. I think not unless we are going to go back and add it everywhere else.

Reply:
I don't know where everywhere else is as I view this document as foundational and standalone. This document resolves into the greater set of documents anyone may read about the TAC governance. The WG R&R should surely have the same note and I think promulgating the Code Conduct far and wide through these types of documents is useful and necessary.